packages/donor/src/screens/bookDonation/BookDonationScreenContainer.tsx

```diff
@@ -69,6 +67,15 @@ export function BookDonationScreenContainer({
     }
   };
 
+  const onSlotSelectedPopUpProps = {
+    open: showWarningPopup,
+    onApproved: continueDonation,
+    onBack: () => {
+      setShowWarningPopup(false);
+      navigate(MainNavigationKeys.BookDonation);
```

Comment:
Why do we need to navigate? Aren't we already in the booking screen?

Reply:
No, the popup should pop before the questionnaire